### ISSUE TITLE
Ext_Probe_copy & lig_probe_copy

### ISSUE THREAD

**Issue by Kapilavasthi6** [shendurelab/MIPGEN#55]
I am ruining MIPGEN with very basic options, but in my output  ....picked_mips.txt file does not contain any number in Ext_Probe_copy & lig_probe_copy getting only 0. 
(../MIPGEN/mipgen -regions_to_scan practice_genes.bed -project_name practice_design -min_capture_size 162 -max_capture_size 162 -bwa_genome_index /<path to genome reference fasta file>/human_g1k_v37.fa)

**Comment by augustboyle** [comment]
Are you sure you're using the right path for your bwa_genome_index? The example should still be able to run.

**Comment by Kapilavasthi6** [comment]
Yes, I downloaded the same file from the UCSC (human_g1k_v37.fa), used with the (mipgen -regions_to_scan Practice_genes.bed -project_name Practice_genes -min_capture_size 38 -max_capture_size 60 -max_mip_overlap 0 -max_arm_copy_product 10 -bwa_genome_index human_g1k_v37.fa
